<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>608</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>923</width>
    <height>608</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>923</width>
    <height>608</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>客户端</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget{
	
	border-image: url(:/Photo/MainWindow/游戏界面.jpg);
}</string>
   </property>
   <widget class="QPushButton" name="AssistButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>155</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#AssistButton{
	image: url(:/Photo/MainWindow/辅助游戏A.png);
}

QPushButton#AssistButton:hover{
	
	image: url(:/Photo/MainWindow/辅助游戏B.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>238</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#BackButton{
	
	
	image: url(:/Photo/MainWindow/辅助游戏返回A.png);
}

QPushButton#BackButton:hover{
	
	
	image: url(:/Photo/MainWindow/辅助游戏返回B.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ManageButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#ManageButton{
	image: url(:/Photo/MainWindow/管理A.png);
}
QPushButton#ManageButton:hover{
	image: url(:/Photo/MainWindow/管理B.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BGButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>181</width>
      <height>39</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#BGButton
{		
	image: url(:/Photo/MainWindow/CS Go-A.png);
}

QPushButton#BGButton:hover
{		
	
	image: url(:/Photo/MainWindow/CS GO-B.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="EEGButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#EEGButton{
	image: url(:/Photo/MainWindow/脑电程序A.png);
}
QPushButton#EEGButton:hover{
	image: url(:/Photo/MainWindow/脑电程序B.png);
}
</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ParticipantButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>460</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>被试选择</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TestButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>FTP测试</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InfoButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Photo/头像.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="LogoutButton">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>310</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#LogoutButton{
	image: url(:/Photo/MainWindow/退出游戏A.png);
}
QPushButton#LogoutButton:hover{
	image: url(:/Photo/MainWindow/退出游戏B.png);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PointerButton">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>160</y>
      <width>97</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#PointerButton{	
	image: url(:/Photo/Game/射击A.png);
}

QPushButton#PointerButton:hover{	
	image: url(:/Photo/Game/射击B.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="VoiceButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>168</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#VoiceButton
{
	image: url(:/Photo/Game/声音匹配A.png);
}

QPushButton#VoiceButton:hover
{
	image: url(:/Photo/Game/声音匹配B.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="FlopButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>105</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#FlopButton{
	
	image: url(:/Photo/Game/翻牌A.png);
}
QPushButton#FlopButton:hover{
	
	image: url(:/Photo/Game/翻牌B.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="HandEyeButton">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>310</y>
      <width>176</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#HandEyeButton{
	
	image: url(:/Photo/Game/手眼协调A.png);
}

QPushButton#HandEyeButton:hover{
	
	image: url(:/Photo/Game/手眼协调B.png);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
